bodhi/server/models.py

```diff
@@ -1748,7 +1748,7 @@ def set_request(self, db, action, username):
             return
 
         # Disable pushing critical path updates for pending releases directly to stable
-        if action is UpdateRequest.stable and self.critpath:
+        if action is (UpdateRequest.stable or UpdateRequest.batched) and self.critpath:
```

Comment:
The ```is``` operator doesn't quite work this way. I believe the above may always evaluate to False because ```UpdateRequest.stable or UpdateRequest.batched``` will evaluate to ```True```, and then ```action is True``` will evaluate to False.

Probably we want this:

```
if action in (UpdateRequest.stable, UpdateRequest.batched) and self.critpath:
```

Or alternatively,

```
if (action is UpdateRequest.stable or action is UpdateRequest.batched) and self.critpath:
```